eitprocessing/eit_data/__init__.py

```diff
@@ -35,27 +32,24 @@ class EITData(SelectByTime, Equivalence, ABC):
     phases: list = field(default_factory=list)
     events: list = field(default_factory=list)
     label: str | None = None
-    variants: VariantCollection = field(
-        default_factory=lambda: VariantCollection(EITDataVariant)
-    )
+    pixel_impedance: NDArray = field(repr=False, kw_only=True)
 
     def __post_init__(self):
         if not self.label:
             self.label = f"{self.__class__.__name__}_{id(self)}"
         self._check_equivalence = ["vendor", "framerate"]
 
     @classmethod
-    def from_path(  # pylint: disable=too-many-arguments,too-many-locals
+    def from_path(
         cls,
         path: PathArg,
         vendor: Vendor | str,
```

Comment:
No, I guess you're correct. I added this because there were different type declarations which were inconsistent with each other. I removed the custom aliases in 8adfceac4c5912da941e84e9a12dbb36c7469f92.